EatGoodNaija.Server/Migrations/20240416063053_r1.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace EatGoodNaija.Server.Migrations
+{
+    public partial class r1 : Migration
```

Comment:
Nothing is added to this migration. What purpose is it serving in the application ?